packages/terra-dialog/docs/README.md

```diff
@@ -1,6 +1,6 @@
 # Terra Dialog
 
-{insert description}
+Wrapper for terra-content-container intended to be consumed from within a modal or popup.
```

Comment:
Let's move away from talking about the terra-content-container, it is an implementation detail and we may move away from using content-container in the future. Instead let's use this verbiage for the description:

```
Dialogs are temporary views that can be used in a myriad of ways. Dialogs have the ability to alert users to act on critical information. In doing so, Dialogs may allow users to avoid destructive decisions, and also extend user workflows without disorienting the user.
```

This is pulled from the Cerner Design Standard entry for dialogs.

Reply:
https://github.com/cerner/terra-core/pull/1173/commits/0e9b0f65bb69c708fa9c6085f050ffd0bc137b6b